src/scripts/main.js

```diff
@@ -354,7 +354,44 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tableBody = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in the DOM. If it doesn't, `tableBody` will be `null`, and attempting to append rows to it will result in an error. Consider adding a check to ensure `tableBody` is not `null` before proceeding with the loop.